<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VNC Session</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Persistence and Visibility</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>The session is visible to others and can be reconnected. It keeps running when you disconnect.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>26</number>
         </property>
         <property name="buddy">
          <cstring>chkPersistencePersistent</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="chkPersistenceNonPersistent">
         <property name="text">
          <string>Non-persistent, private</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="chkPersistencePersistent">
         <property name="text">
          <string>Persistent, visible</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>The session is not visible to others and will be terminated when you disconnect.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>26</number>
         </property>
         <property name="buddy">
          <cstring>chkPersistenceNonPersistent</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Session name</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="editSessionName"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Security</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="chkSecurityUnknown">
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="widgetVncAuth" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>35</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="buddy">
             <cstring>editPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editPassword">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>View-only password:</string>
            </property>
            <property name="buddy">
             <cstring>editViewonlyPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editViewonlyPassword">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="chkSecurityNone">
         <property name="text">
          <string>No password required</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="chkSecurityVncAuth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Require custom password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="chkSecurityPlain">
         <property name="text">
          <string>Require user login</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Allowed users:</string>
            </property>
            <property name="indent">
             <number>26</number>
            </property>
            <property name="buddy">
             <cstring>editAllowedUsers</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editAllowedUsers">
            <property name="toolTip">
             <string>List of usernames separated by commas.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Sharing</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QRadioButton" name="chkSharingUnknown">
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="chkSharingMultipleClients">
         <property name="text">
          <string>Allow multiple clients at time</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="chkSharingOneClient">
         <property name="text">
          <string>Allow one client at time</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chkPersistenceNonPersistent</tabstop>
  <tabstop>chkPersistencePersistent</tabstop>
  <tabstop>editSessionName</tabstop>
  <tabstop>chkSecurityUnknown</tabstop>
  <tabstop>chkSecurityNone</tabstop>
  <tabstop>chkSecurityVncAuth</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editViewonlyPassword</tabstop>
  <tabstop>chkSecurityPlain</tabstop>
  <tabstop>editAllowedUsers</tabstop>
  <tabstop>chkSharingUnknown</tabstop>
  <tabstop>chkSharingOneClient</tabstop>
  <tabstop>chkSharingMultipleClients</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkPersistencePersistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPersistencePersistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPersistencePersistent</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSecurityPlain</sender>
   <signal>toggled(bool)</signal>
   <receiver>editAllowedUsers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSecurityVncAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>editPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSecurityVncAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>editViewonlyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
